SUMMARY.md

```diff
@@ -0,0 +1,24 @@
+# Summary
```

Comment:
I don't see any links to this file. Will it be used by the gitbook?

Reply:
Yes, if you do a `npm run build:watch`, you can see the `SUMMARY.md` file in use in the left-hand side:

![screen-shot](https://cl.ly/32212r1J3X2K/download/Image%202018-01-24%20at%2014.06.07.png)

Reply:
looks like redux made SUMMARY.md a symlink to docs/README.md. I kind of like that since it keeps all of the docs in one folder. Do you mind doing that too?